docs/configdoc.md

```diff
@@ -75,24 +79,29 @@ environment: PROD
 |command|`"tc_root"`|Command to run tc_root program| Yes |
 |version|`"1.0"`|Version of tc_root program| Yes |
-|user-program-daemon|`"false"`|Set to true it requires l3afd to stop the application (via SIGTERM on Linux or SIGKILL on Windows)| Yes |
+
+## [ebpf-chain-debug]
+| FieldName | Example            | Description                                                    | Required |
+|-----------|--------------------|----------------------------------------------------------------|----------|
+| addr      | `"localhost:8899"` | Hostname and Port of chaining debug REST API                   | No       |
+| enabled   | `"true"`           | Boolean to check ebpf chaining debug details is enabled or not | No       |
 
 ## [l3af-configs]
-| FieldName     | Example       | Description     | Required        |
-| ------------- | ------------- | --------------- | --------------- |
-|restapi-addr|`"localhost:53000"`| Hostname and Port of l3af-configs REST API | Yes |
+| FieldName     | Example       | Description     | Required |
+| ------------- | ------------- | --------------- |----------|
+|restapi-addr|`"localhost:53000"`| Hostname and Port of l3af-configs REST API | No       |
 
 # [l3af-config-store]
 | FieldName     | Example       | Description     | Required        |
 | ------------- | ------------- | --------------- | --------------- |
 |filename|`"/etc/l3afd/l3af-config.json"`|Absolute path of persistent config file where we are storing L3afBPFPrograms objects. For more info see [models](https://github.com/l3af-project/l3afd/blob/main/models/l3afd.go)| Yes |
 
 # [mtls]
-| FieldName     | Example       | Description     | Required        |
-| ------------- | ------------- | --------------- | --------------- |
-|enabled| `"true"` | Boolean to check mtls enabled or not on REST API exposed by l3afd| Yes |
-|min-tls-version|`"1.3"`| Minimum tls version allowed| No |
-|cert-dir|`"/etc/l3af/certs"`|Absolute path of ca certificates. On Linux this points to a filesystem directory, but on Windows it can point to [certificate store](https://docs.microsoft.com/en-us/windows-hardware/drivers/install/certificate-stores) | Yes |
-|server-crt-filename|`"server.crt"`|Server's ca certificate filename| Yes |
-|server-key-filename|`"server.key"`|Server's mtls key filename| Yes |
-|cert-expiry-warning-days|`"30"`|How many days before expiry you want warning| No |
\ No newline at end of file
+| FieldName     | Example       | Description     | Required |
+| ------------- | ------------- | --------------- |----------|
+|enabled| `"true"` | Boolean to check mtls enabled or not on REST API exposed by l3afd| No       |
+|min-tls-version|`"1.3"`| Minimum tls version allowed| No       |
+|cert-dir|`"/etc/l3af/certs"`|Absolute path of ca certificates. On Linux this points to a filesystem directory, but on Windows it can point to [certificate store](https://docs.microsoft.com/en-us/windows-hardware/drivers/install/certificate-stores) | No       |
```

Comment:
```suggestion
|cert-dir|`"/etc/l3af/certs"`|Absolute path of CA certificates. On Linux this points to a filesystem directory, but on Windows it can point to a [certificate store](https://docs.microsoft.com/en-us/windows-hardware/drivers/install/certificate-stores) | No       |
```